docs/topics/DataWorkerService.md

```diff
@@ -1,5 +1,59 @@
 # DataWorkerService
 
-These docs are currently under construction!
+The [DataWorkerService](https://github.com/osu-tournament-rating/otr-api/tree/master/DataWorkerService) is a program which is part of
+the [](o-TR-API.md) repository. This program is a service which continuously polls the database for items which need to be processed. This includes the following processes:
 
-<toc depth="10" />
\ No newline at end of file
+* Fetching match data from the [osu! API](https://osu.ppy.sh/docs/index.html)
+* Fetching historical player data from the [osu!track API](https://github.com/Ameobea/osutrack-api)
+* Running [automated checks](Automated-Checks.md) against tournament data
+* Performing stat calculations, such as match cost and forming placements (required by the [](o-TR-Processor.md)).
+
+## Core Principles
+
+This system is designed with the following principals in mind:
```

Comment:
```suggestion
This system is designed with the following principles in mind:
```
how did this change LOL it was correct in the other file

Reply:
:ehehe: